template.yaml

```diff
@@ -8,7 +8,7 @@ Description: >
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 10
+    Timeout: 30
```

Comment:
this is irrelevant here, the timeout should be changed in the deployment

Reply:
I've increased it for invoking locally using `sam local invoke`